src/QsCompiler/DataStructures/Diagnostics.fs

```diff
@@ -457,614 +457,631 @@ type DiagnosticItem =
 
         try
             String.Format(str, args)
-        with _ -> str // let's fail silently for now
+        with
+        | _ -> str // let's fail silently for now
 
     static member Message(code: ErrorCode, args: IEnumerable<string>) =
         let ApplyArguments =
             DiagnosticItem.ApplyArguments args
             << function
```

Comment:
I think you could avoid most of the formatting changes in this file by using `match` here and in the other cases:

```suggestion
            match DiagnosticItem.ApplyArguments args with
```

Reply:
I'll try that and see how the diff compares...

Reply:
Yup, that worked, thanks!

Reply:
Oh, I made a type error... I think you have to add a parameter:

```fsharp
let ApplyArguments code =
    match DiagnosticItem.ApplyArguments args code with
...
```
```

Reply:
oh, hmm... I think it still has a problem, and I need to have a `<<` somewhere. Let me see if I can figure it out, but it may end up that the large Fantomas update to the indentation is not avoidable...

Reply:
I think we've found a form that works and still minimizes the diff. Thanks!